homeassistant/helpers/selector.py

```diff
@@ -222,7 +290,11 @@ class EntitySelector(Selector):
     selector_type = "entity"
 
     CONFIG_SCHEMA = SINGLE_ENTITY_SELECTOR_CONFIG_SCHEMA.extend(
-        {vol.Optional("multiple", default=False): cv.boolean}
+        {
+            vol.Optional("exclude_entities"): [str],
+            vol.Optional("include_entities"): [str],
```

Comment:
Update the validation to ensure `exclude_entities` are not present.

Reply:
Should we also check that only entities in `include_entities` are present?
Are `include_entities` + `exclude_entities` supported both in multiple and single mode?

Reply:
yes and yes